docs/source/developer/initial_setup.rst

```diff
@@ -0,0 +1,81 @@
+**************************************
+Setting Up CMakePPLang for Development
+**************************************
+
+This page contains information to help developers new to CMakePPLang set up
+the project to begin development.
+
+Downloading the Source Code
+===========================
+
+During development, the latest version of CMakePPLang should be used.
+CMakePPLang is `hosted on GitHub <CMakePPLang_src_>`_ and can be cloned to
+your local system using Git_:
+
+.. code-block:: bash
+
+   git clone https://github.com/CMakePP/CMakePPLang.git
+
+Building CMakePPLang
+====================
+
+CMakePPLang is built on top of and extends the `CMake language <CMake_>`_, so
+"building" CMakePPLang is performed during CMake's configuration step. The
+build step usually required by projects using CMake is not required to
+build CMakePPLang itself, so it will not be included here.
```

Comment:
Maybe move the "The build step..." sentence to the end of the section to avoid confusion with forthcoming commands.